### ISSUE TITLE
Automatically hiding taskbar doesn't work properly on second monitor

### ISSUE THREAD

**Issue by aberdean** [jetspiking/Taskbar11#20]
When selecting the option to automatically hide the taskbar, it works perfectly on the first monitor. Moving the mouse to the top or bottom of the screen (depending on the selected taskbar position) displays the taskbar and then it automatically hides as the mouse is moved away.
However, on the second monitor it shows up only the first time you move the mouse towards it. Then, it hides as the mouse is moved away and it never shows up again, no matter how many time you may try to move the mouse in its direction. 
Note also that if you move the mouse toward the taskbar first on the first monitor, then on the second one it would never show up, not even the first time you move the mouse towards it.

**Comment by jetspiking** [comment]
Thank you for reporting this issue! I have confirmed the problem. I have tried various settings in Windows to attempt to fix this (outside of Taskbar 11). This might be a bug in the Windows 11 UI because without involving Taskbar11 I was unable to get it working.

Currently I do not have a solution, sorry.

**Comment by aberdean** [comment]
Makes sense. There was actually the same problem in Windows 10 when they just released it. Then, they had fixed it.
Probably, they forgot the fix in the meanwhile and now it's back in Windows 11.
Let's hope they'll fix it again.

Thank you for taking the time to look into it.

**Comment by aberdean** [comment]
I just found out something that may help you investigate this issue.

Try this: 
1. Set the taskbar to the top using Taskbar 11
2. Set automatically hide the taskbar from Windows settings

Now, it works perfectly.

May be worth investigating the differences between hiding from settings (after using Taskbar 11 to set the position) and hiding from Taskbar 11.

Great job on this Taskbar 11, by the way! Thank you for it and keep going!

**Comment by jetspiking** [comment]
I can confirm something odd seems to be going on with this setting. After adjusting the settings from within Windows the "Equal Taskbar Positions" button in Taskbar11 is magically working again.

However, I can not seem to get the automatically hide option to work, even when following your steps.

**Comment by jetspiking** [comment]
@aberdean Hi there! Could you verify whether the issue is still present in the latest version of Taskbar11?

Sorry for the delayed response.

**Comment by aberdean** [comment]
@jetspiking Apologies, I just seen this.
I tried the latest version and now neither of the taskbars hide at all.
I tried to change settings on Windows as well and no matter what, both taskbars stay put.

**Comment by jetspiking** [comment]
@aberdean Hi, no problem!

I have no idea why this problem would occur. I am using the latest version of Taskbar11 on 22000.556 and can hide the taskbar. Which Windows 11 version are you currently on ('winver' can show the exact build number)?

Are you using other tools that might interfere with the Windows 11 settings? 

Especially through the Windows 11 settings, it should always change back.

**Comment by aberdean** [comment]
@jetspiking Hi, I was in a rush and haven't been specific, sorry.

After a past update of Windows, I wasn't able to hide the taskbar anymore, so the problem is likely to be related to Windows, not to Taskbar11.

After reading your message, I downloaded the latest version of Taskbar11 and nothing changed.
I am a couple of updates behind, though. If I have time later today, I'll update Windows and try again.

I'll let you know.

**Comment by aberdean** [comment]
@jetspiking I'm sorry, I still couldn't update Windows. I'll make a note to get back to you as soon as I do.

**Comment by jetspiking** [comment]
Hi @aberdean,

No worries, take your time.

**Comment by aberdean** [comment]
The only annoyance, which is also kinda weird, is that if I have some windows open on both monitors, say Chrome, some windows on one monitor and some on the other, then if I click on the Chrome icon from the second monitor, it opens the taskbar also on the first monitor. Not the other way around, though. If I click on the first monitor, it doesn't open that on the second monitor.

Usually, it's a minor problem, because then both hide regularly. Sometimes, it can be annoying, depending on what you're doing on the first monitor.
